perception/tensorrt_yolo/README.md

```diff
@@ -8,6 +8,8 @@ This package detects 2D bounding boxes for target objects e.g., cars, trucks, bi
 
 ### Cite
 
+<!-- cspell:disable -->
```

Comment:
Can you use `cspell:ignore word1, word2` here instead?

Reply:
Ok !https://github.com/autowarefoundation/autoware.universe/pull/950/commits/6b7badefdd7e7e314540bfb984f089c7f2cfa93a